packages/notifications/examples/basic.md

```diff
@@ -0,0 +1,269 @@
+### Alert
+
+All usages of `<Close />` must contain an `aria-label` or other assistive technique to have
+discernible text.
+
+```jsx
+const { Field, Label, Radio, Toggle } = require('@zendeskgarden/react-forms/src');
+const { MD } = require('@zendeskgarden/react-typography/src');
+
+const StyledMd = styled(MD)`
+  ${props => `
+    color: ${props.theme.colors.foreground};
+    font-weight: ${props.theme.fontWeights.semibold}
+  `}
+`;
+
+const alertTitles = {
+  success: 'Success Alert',
+  error: 'Error Alert',
+  warning: 'Warning Alert',
+  info: 'Info Alert'
+};
+
+const initialState = { type: 'success', showClose: true };
```

Comment:
@jzempel @allisonacs 

> ...let's drop the "multi-line" knob. It's super confusing with respect to the API which is "use a <Paragraph> if your text spans multiple lines, otherwise just drop in plain text". 

Does the same apply to the `<Alert>` and its adjacent child component `<Close>` too? There isn't a close related prop for `<Alert>`. Similar to how `<Notification>` doesn't have a multi-line prop. `close` and `multi-line` aren't on prop sheets.

Maybe I need to reconsider what a knob means for Garden's examples — knobs don't necessarily map to a prop on the prop sheet. Instead, they are form elements that dynamically show possible renders, which include different usages in combination with various child components (e.g., rendering a `<Paragraph>` within a `<Notification>` or `<Close>`  within a `<Alert> `.

It felt strange to not have a knob for multi-line for `<Notification>`, but have a show close knob for `<Alert>`. I added a knob for both, but renamed their labels to "Show Close" and "Show Multi-line":

![Examples](https://user-images.githubusercontent.com/1811365/70739146-08cfdb00-1ccb-11ea-928c-9f669032bb85.gif)

Reply:
@allisonacs and I walked through the UI that was recorded above. We both think it's sufficient. @jzempel, I know we discussed removing the multi-line toggle, but I added the multi-line toggle back so that there's consistency with `<Alert>`'s "Show Close" toggle. It felt strange having a "Show Close", but not having a "Show Multi-line".